admin-portal/src/pages/editPledge/components/pledgeSummary/pledgeSummary.tsx

```diff
@@ -1,15 +1,30 @@
 import React, { ChangeEvent } from "react";
 import "./pledgeSummary.css";
 import { Pledge } from "../../../../types/Pledge";
+import { AidPackage } from "../../../../types/AidPackage";
 
 interface PledgeSummaryProps {
   pledge: Pledge;
   onStatusChange: (status: Pledge.Status) => void;
+  aidPackageStatus: AidPackage.Status;
 }
 
+const statusAllowPledgeStatus = {
```

Comment:
```suggestion
const pledgeStatusToAllowedStatuses = {
```

Also add the data type